databases/week1/queries.sql

```diff
@@ -0,0 +1,37 @@
+/*Find out how many tasks are in the task table*/
+SELECT COUNT(*) FROM task;
+/*Find out how many tasks in the task table do not have a valid due date*/
+SELECT * FROM task WHERE due_date IS NULL;
+/*Find all the tasks that are marked as done*/
+SELECT * FROM task WHERE status_id = 3;
```

Comment:
you are assuming Done has id 3, but you can also get it by the title

Reply:
Yes, we can do that also. Thank you for your suggestion.